conf/messages

```diff
@@ -368,7 +368,8 @@ page.reg.registrationUnsuccessful.whatHappensNext.p3.part3          = GOV.UK pag
 page.reg.cancelservice.warning                                      = All the information you''ve provided will be deleted.
```

Comment:
Since you're not re-using this one, could you remove it? :recycle:

Reply:
its being used